packages/sui-domain/src/AnemicModel.js

```diff
@@ -1,3 +1,28 @@
+/**
+ * Recursively calls toJSON on each member
+ * @param {Object} obj
+ */
+const toJSON = obj => {
+  Object.keys(obj).forEach(key => {
+    obj[key] = obj[key] instanceof AnemicModel ? obj[key].toJSON() : obj[key]
+    obj[key] = obj[key] instanceof Object ? toJSON(obj[key]) : obj[key]
+  })
+  return obj
+}
+
+/**
+ * Converts anemic instance to an object with public props
+ * @param {AnemicModel} obj
+ */
+const anemicInstanceToObject = obj =>
+  Object.keys(obj)
+    .map(key => key.replace('_', ''))
```

Comment:
I'm a bit concern to loop more times but this code is cleaner this way.
I can change it to a unique loop

Reply:
Yup it iterates more but performance here doesn't seems to be a problem.... 👌